modules/cluster-admin/providers.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     confluent = {
       source  = "confluentinc/confluent"
-      version = ">= 1.2.0"
+      version = "~>1.0.0"
```

Comment:
This will pin the version to only `v1.0.x` providers, which we do not want. You will want `~> 1.0`. Also please include the space.